sensitivity.tex

```diff
@@ -0,0 +1,126 @@
+    \label{fig:br_rep}
+\end{figure}
+
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/br_uox.png}
+    \end{center}
+    \caption{Increasing the breeding ratio decreases the number of total \gls{UOX} \gls{UNF}
+             since the \gls{ASTRID} becomes self-sustaining earlier, no longer requiring
+             plutonium from reprocessing \gls{UOX}.}
+    \label{fig:eu_tail}
+\end{figure}
+
+
+\subsection{Lifetime Extension of French \glspl{LWR}}
+Extending the lifetime of French \glspl{LWR} dramatically lowers the average
+monthly \gls{UOX} reprocessing demand, since the \gls{ASTRID} deployment becomes less
+aggressive. the plutonium demand is delayed and allows the reprocessing plant more
+time to produce the same amount of plutonium for \gls{ASTRID} reactors. The
+extended deployment schemes are illustrated in figure \ref{fig:ext}. 
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/ext_uox.png}
+    \end{center}
+    \caption{Increasing the lifetime of French \glspl{LWR} decreases the monthly
+             average \gls{UOX} reprocessing demand.}
+    \label{fig:ext}
+\end{figure}
+
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/ext_all.png}
+    \end{center}
+    \caption{The shift in \gls{ASTRID} deployment has little impact on the total monthly
+             reprocessing capacity.}
+    \label{fig:ext}
+\end{figure}
+
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[height=0.25\textheight]{./images/sensitivity/5.png}
+        \includegraphics[height=0.25\textheight]{./images/sensitivity/10.png}
+        \includegraphics[height=0.25\textheight]{./images/sensitivity/20.png}
+    \end{center}
+    \caption{The shift in \gls{ASTRID} deployment allows more time for the reprocessing
+             plant to prepare the plutonium for \gls{ASTRID} fuel production, lowering
+             the average monthly reprocessing capacity.}
+    \label{fig:ext}
+\end{figure}
+
+\iffalse
+\begin{figure}[htbp]
+    \centering
+    \begin{subfigure}[b]{0.4\textwidth}
+        \includegraphics[width=\textwidth]{./images/sensitivity/5.png}
+        \caption{5 years}
+    \end{subfigure}
+    \begin{subfigure}[b]{0.4\textwidth}
+        \includegraphics[width=\textwidth]{./images/sensitivity/10.png}
+        \caption{10 years}
+    \end{subfigure}
+    \begin{subfigure}[b]{0.4\textwidth}
+        \includegraphics[width=\textwidth]{./images/sensitivity/20.png}
+        \caption{20 years}
+    \end{subfigure}
+    \caption{The shift in \gls{ASTRID} deployment allows more time for the reprocessing
+             plant to prepare the plutonium for \gls{ASTRID} fuel production, lowering
+             the average monthly reprocessing capacity.}
+    \label{fig:ext_hor}
+\end{figure}
+\fi
+
+The lifetime extension of French \glspl{LWR} have little impact on the total
```

Comment:
"The lifetime extension... have" is incorrect. This is a subject verb plurality mismatch.